docs/source/system_reference_guide/oss_documentation/maap_zenodo.ipynb

```diff
@@ -0,0 +1,158 @@
+{
```

Comment:
<p>This is not the recommended workflow for "Algorithms" that are in github. Also this doesn't address that users could upload to Zenodo and then submit to a Community since they can have items belong to more than one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Thanks! I've added where users can upload to Zenodo and then submit to a community. What is the recommended workflow for "Algorithms" in Github?</p>

Reply:
Use the Zenodo Github import, then submit to the community. This is recommended because all future versions (releases in github) will automatically receive DOIs.
https://docs.github.com/en/repositories/archiving-a-github-repository/referencing-and-citing-content